packages/catalog-server/src/lib/firestore/firestore-repository.ts

```diff
@@ -219,23 +220,28 @@ export class FirestoreRepository implements Repository {
       const versionDistTags = getDistTagsForVersion(distTags, version);
 
       // Store package data and mark version as ready
-      await t.set(versionRef, {
+      // TODO: make converter handle denormalized data
```

Comment:
what is denormalized data in this context?

Reply:
The fields copied from PackageInfo like the packageName and distTags... This is actually already handled by those fields being in the GraphQL schema. Not sure fi we need to do more, so I'll remove the comment.